### ISSUE TITLE
Review revised "headings" on homepage

### ISSUE THREAD

**Issue by kimisgold** [omeka/theme-thanksroy#56]
Previously, the homepage skipped the level 1 heading, going straight to using `h2` and `h3` elements for the featured records. These headings didn't actually feel appropriate for their content, so I replaced them with divs that use the heading presentation, but feel more semantically accurate. I also condensed record links to include both images and their titles to avoid repetitive links.

Please test the following:

* There are no skipped headings on the homepage.
* Featured records and recent items have similar presentation and appropriate styles.
* All record links contain both the item and image.

These should be tested on both desktop and mobile browsers.

**Comment by katknow** [comment]
This seems to be working well across browsers and on mobile (developer tools & on my iPhone)
